.github/workflows/pr-test-suite.yml

```diff
@@ -1226,6 +1261,14 @@ jobs:
       python-version: '3.9'
     steps:
     - uses: actions/checkout@v4
+
+    # Install protobuf compiler for Lance dependency
+    - name: Install protobuf compiler
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y protobuf-compiler
+        protoc --version
```

Comment:
Rather than making protoc a build dependency, we currently favor checking in pre-build protos.

The [daft-proto package has an example of using `.build.rs`](https://github.com/Eventual-Inc/Daft/tree/main/src/daft-proto) and a [trick to change the name](https://github.com/Eventual-Inc/Daft/blob/main/Makefile#L99-L101) to `build.rs` and back to generate.